### PR TITLE
:bug: some typos

### PR DESCRIPTION
**Pull request by beseven**

### CODE REVIEW

**Comment by c2s-dev** [comment]
:rocket: deployed to [http://profiles-pr-39.dev.c2s.nhschoices.net](http://profiles-pr-39.dev.c2s.nhschoices.net)

File: app/views/book-appointment.nunjucks [hunk 1]
```diff
@@ -73,30 +81,30 @@
   <table>
     <tr>
       <td>Monday</td>
-      <td>7:30am to 8pm</td>
+      <td>7.30am to 8pm</td>
```
Comment (st3v3nhunt) [inline app/views/book-appointment.nunjucks:84]: Is this changing to adhere to a specific style guide?

Comment (beseven) [reply app/views/book-appointment.nunjucks:84]: yes, Dean said this the beta styleguide related to timings.

**Review by st3v3nhunt — APPROVED** [review]
LGTM 👍

**Pull request merged**